src/myst.ts

```diff
@@ -48,11 +70,8 @@ const evalRole: RoleSpec = {
   }
 };
 
-export function markdownParse(text: string): Root {
-  const mdast = mystParse(text, {
-    directives: [cardDirective, gridDirective, ...tabDirectives],
-    roles: [evalRole]
-  });
+export function markdownParse(text: string, options: Partial<AllOptions>): Root {
```

Comment:
Should `options` be potentially undefined here (and other places)?

```suggestion
export function markdownParse(text: string, options?: Partial<AllOptions>): Root {
```

Reply:
I am newish to typescript, actually I didn't know that was available syntax.... but anyway, I don't know? Why?